src/lib/common/statistics.h

```diff
@@ -38,7 +38,7 @@
   struct timespec renderStart;                                         \
   struct timespec renderEnd;                                           \
                                                                        \
-  if (reqTimeStatistics)                                               \
+  if (timingStatistics)                                               \
   {                                                                    \
```

Comment:
The backslashes were perfectly indented before ...
[ Sorry for being such a petty asshole ;-) ]